src/inputs/WithLabel/Label.tsx

```diff
@@ -1,15 +1,17 @@
 import React from 'react';
+import clsx from 'clsx';
 import { Text } from 'data';
 
 export type LabelProps = {
   value: React.ReactNode;
+  className: string;
```

Comment:
Make this optional as you might not always need to pass classNames :D 
```suggestion
  className?: string;
```